.github/workflows/be-dev-ci.yml

```
@@ -28,22 +28,61 @@ jobs:
+
+    outputs:
+      dependencies-changed: ${{ steps.filter.outputs.dependencies }}
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./backend
+
+    steps:
+      - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)